controllers/comments.controllers.js

```diff
@@ -29,3 +29,19 @@ exports.postNewCommentToArticleId = (req, res, next) => {
             res.status(201).send({ newComment });
         }).catch(next);
 };
+
+exports.deleteCommentById = (req, res, next) => {
+    const commentId = req.params.comment_id;
+    console.log("🚀 ~ commentId:", commentId);
+    const promises = [removeComment(commentId),];
+
+    if (commentId) {
+        promises.push(getCommentById(commentId));
+    } else {
+        return res.send(400).send({ msg: "bad request" });
+    }
```

Comment:
In order for this endpoint to be accessed there must be something the path can use for the _comment_id_ , so I don't think the `if` here would ever be false. In which case, you could bypass the need to `push` and have the function `getCommentById` inside the array on line _36_ as a second element